libkbfs/init.go

```diff
@@ -158,7 +167,9 @@ func DefaultInitParams(ctx Context) InitParams {
 		},
 		TLFJournalBackgroundWorkStatus: TLFJournalBackgroundWorkEnabled,
 		StorageRoot:                    ctx.GetDataDir(),
-		Mode:                           InitDefaultString,
+		BGFlushPeriod:                  1 * time.Second,
```

Comment:
Did you mean to use `bgFlushPeriodDefault` here? (also the `BGFlushDirOpBatchSize` below)